cocos/core/pipeline/deferred/postprocess-stage.ts

```diff
@@ -167,25 +158,18 @@ export class PostprocessStage extends RenderStage {
         cmdBuff.bindDescriptorSet(SetIndex.GLOBAL, pipeline.descriptorSet);
 
         // Postprocess
-        let pass: Pass;
-        let shader: Shader;
-        const builtinPostProcess = builtinResMgr.get<Material>('builtin-post-process-material');
-        if (builtinPostProcess) {
-            pass = builtinPostProcess.passes[0];
-            shader = ShaderPool.get(pass.getShaderVariant());
-        } else {
-            pass = this._postprocessMaterial!.passes[POSTPROCESSPASS_INDEX];
-            shader = ShaderPool.get(this._postprocessMaterial!.passes[POSTPROCESSPASS_INDEX].getShaderVariant());
-        }
+        const builtinPostProcess = sceneData.deferredPostMat;
+        const pass = builtinPostProcess.passes[0];
+        const shader = ShaderPool.get(pass.getShaderVariant());
```

Comment:
Postprocess stage should bind material descriptorSet too.

```js
cmdBuff.bindDescriptorSet(SetIndex.MATERIAL, pass.descriptorSet);
```

Reply:
Already added it